alcs-api/src/common/authorization/authorization.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Get, Inject, Query, Res } from '@nestjs/common';
+import { FastifyReply } from 'fastify';
+import { Public } from 'nest-keycloak-connect';
+import { CONFIG_TOKEN, IConfig } from '../config/config.module';
+import { AuthorizationService } from './authorization.service';
+
+@Controller('/authorize')
+export class AuthorizationController {
+  constructor(
+    private authorizationService: AuthorizationService,
+    @Inject(CONFIG_TOKEN) private config: IConfig,
+  ) {}
+
+  @Get()
+  @Public()
+  async handleAuth(
+    @Query('code') authCode: string,
+    @Query('session_state') sessionId: string,
+    @Res() res: FastifyReply,
+  ) {
+    try {
+      const token = await this.authorizationService.exchangeCodeForToken(
+        authCode,
+      );
+
+      //TODO: Set in Redis
```

Comment:
I thought we agreed to use local docker redis or is it in next PR?

Reply:
I plan to do the token storing and refresh in the next PR, I figured this PR already was long enough.